pages/facts.html

```diff
@@ -4,9 +4,84 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Friends Fun Facts</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link rel="stylesheet" href="../styles/facts.css">
 </head>
 <body>
-  
+  <header>
+
+    <nav>
+      <ul id="header-links">
+        <li><a href="../index.html">Home</a></li>
+        <li><a href="facts.html">Facts</a></li>
+        <li><a href="gallery.html">Gallery</a></li>
+      </ul>
+    </nav>
+    <div>
+      <img id="friends-logo" src="../images/Friends_logo.svg" alt="Friends Logo">
+      <h1 id="subtitle">
+        F<span>&middot;</span>a<span>&middot;</span>n<span>&middot;</span>s<span>&middot;</span>i<span>&middot;</span>t<span>&middot;</span>e
+      </h1>
+    </div>
+  </header>
+  <main>
+    <section>
+      <div class="heading">
+        <img id="smiley-face" src="../images/smiley.jpg" alt="smiley">
+        <h2>5 <span>&middot;</span> Fun <span>&middot;</span> Friends <span>&middot;</span> TV <span>&middot;</span> Show <span>&middot;</span> Facts<span>&middot;</span>  You <span>&middot;</span> Never<span>&middot;</span>  Knew</h2>
+      </div>
```

Comment:
Same comment here about using spans for styling the dots. 

Instead of spans, you could use an svg and have some alt text like "5 Fun Friends TV Show Facts You Never Knew". 

If you did want to keep this HTML, then instead of using an H2 tag, I'd probably do something like:

```HTML
<div>
  <p class="facts-title">5</p>
  <span class="fansite-dot">&middot;</span>
  <p class="facts-title">Fun</p>
  <span class="fansite-dot">&middot;</span>
</div>